src/app-layer-ftp.c

```diff
@@ -152,6 +153,16 @@ static void FTPParseMemcap(void)
 
     SC_ATOMIC_INIT(ftp_memuse);
     SC_ATOMIC_INIT(ftp_memcap);
+
+    if ((ConfGet("app-layer.protocols.ftp.maxtx", &conf_val)) == 1) {
```

Comment:
`maxtx` here

Reply:
Thanks for spotting this, should be `max-tx` indeed